packages/types/src/event-bus/index.ts

```diff
@@ -0,0 +1,24 @@
+export type Subscriber<T = unknown> = (
```

Comment:
self-review: @adrien2p + @carlos-r-l-rodrigues Can I get you to validate my structuring of types and utils in `@medusajs/types` and `@medusajs/utils`? :)

Reply:
nice 💪🏻